build/configs/default_config.yaml

```diff
@@ -152,7 +152,8 @@ ethereum:
     nftMint: 900000
     nftTransferFrom: 150000
   # Timeout to wait for an ethereum transaction to be added to a block and events triggered
-  contextWaitTimeout: "600s"
+  # Set to higher bound ~28 days
+  contextWaitTimeout: "672h"
```

Comment:
Since go-celery max limit is the same, makes sense to keep it here, since they go hand by hand